app/graphql/mutations/update_submission_mutation.rb

```diff
@@ -32,6 +32,10 @@ class UpdateSubmissionMutation < Mutations::BaseMutation
     argument :provenance, String, required: false
     argument :signature, Boolean, required: false
     argument :state, Types::StateType, required: false
+    argument :sourceArtworkID,
+             String,
+             required: false,
+             description: 'If this artwork exists in Gravity, its ID'
```

Comment:
Why do you want to add the field to the UpdateSubmissionMutation? I suppose it should be enough to just initialize the value on submission creation. It's actual also for `my_collection_artwork_id`

Reply:
Have you tried to move `sourceArtworkID` directly to the submission create call as it happens for the `id` field on submission update?
![image](https://user-images.githubusercontent.com/79979820/158211579-bd024f3b-4f1a-43c5-887d-20efec864736.png)

Reply:
This works 🌟 I'll close this PR and go with your solution.